pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -123,32 +132,71 @@ func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEventWithHint {
 	return events
 }
 
+func (pl *VolumeBinding) isSchedulableAfterStorageClassChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
+	oldSC, newSC, err := util.As[*storagev1.StorageClass](oldObj, newObj)
+	if err != nil {
+		return framework.Queue, err
+	}
+
+	logger = klog.LoggerWithValues(
+		logger,
+		"Pod", klog.KObj(pod),
+		"StorageClass", klog.KObj(newSC),
+	)
+
+	pinfoList := getPVCInfoListFromPodVolumes(pod)
+	for _, pinfo := range pinfoList {
+		pvc, err := pl.PVCLister.PersistentVolumeClaims(pod.Namespace).Get(pinfo.pvcName)
+		if err != nil {
+			if pinfo.isEphemeral && apierrors.IsNotFound(err) {
+				err = fmt.Errorf("waiting for ephemeral volume controller to create the persistentvolumeclaim %q", pinfo.pvcName)
+				return framework.Queue, err
```

Comment:
Why return an error? If so, the pod will put into BackOffQ. It doesn't make pod become schedulable.

Reply:
I've tried committing the code content mentioned in the above comment.